static/espers.css

```diff
@@ -120,9 +120,10 @@
 
 #gridContainer {
     position: relative;
-    width:1000px;
-    height:1000px;
-    transform-origin:50% 50%;
+    width: 1000px;
+    height: 1000px;
+    transform-origin: 50% 50%;
+    z-index: -1;
```

Comment:
Why was this z-index needed ? Unable to select esper node with it, I removed it.

Reply:
Ah damn... It was for the box-shadow to stay in front of the inner node... ☹️